src/panels/lovelace/editor/card-editor/hui-card-picker.ts

```diff
@@ -299,24 +299,27 @@ export class HuiCardPicker extends LitElement {
       }
     }
 
+    const cardName = customCard
+      ? `${this.hass!.localize(
+          "ui.panel.lovelace.editor.cardpicker.custom_card"
+        )}: ${customCard.name || customCard.type}`
+      : name;
+
     return html`
       <div class="card">
-        <div
+        <button
           class="overlay"
+          type="button"
```

Comment:
Either remove here or add above for consistency

Reply:
I added it on the other button